pkg/logging/constants.go

```diff
@@ -40,6 +40,8 @@ const (
 	NamespaceKey = "namespace"
 	// NameKey is used to specify a name when a log is related to an object.
 	NameKey = "name"
+	// APIVersionKey is used to specify an API version when a log is related to an object.
+	APIVersionKey = "apiVersion"
```

Comment:
I think we short find short keys for the common ones. `ws`, `ns`, `v`

Reply:
I generally disagree ;)